R/format_vpc_input_data.R

```diff
@@ -62,7 +62,7 @@ format_vpc_input_data <- function(dat, cols, lloq, uloq, stratify, log_y, log_y_
   dat <- standardize_column(dat=dat, cols=cols, colname="id", coldesc="id indicator", what=what)
   dat <- standardize_column(dat=dat, cols=cols, colname="dv", coldesc="dependent variable", what=what)
   dat <- standardize_column(dat=dat, cols=cols, colname="idv", coldesc="indepentent variable", what=what)
-  if (!pred_corr) {
+  if (pred_corr) {
```

Comment:
Our (small amount of) discussion on it is in #77.  The warning says that "VPC plot will be shown for non-censored data only!" which suggests that the VPC plot should omit censored data.  The `!pred_corr` behavior would show a value at the LOQ for the censored values rather than not showing them which seems to go against the message in `vpc.R`.

Whichever way is correct, adding some comments to the code may help to prevent future edits that move to incorrect behavior.

Reply:
Our (small amount of) discussion on it is in #77.  The warning says that "VPC plot will be shown for non-censored data only!" which suggests that the VPC plot should omit censored data.  The `!pred_corr` behavior would show a value at the LOQ for the censored values rather than not showing them which seems to go against the message in `vpc.R`.

Whichever way is correct, adding some comments to the code may help to prevent future edits that move to incorrect behavior.

Reply:
Ah I see... thanks for refreshing my mind. 

I think the best way is to not have that if-statement at all, and filter out (i.e. set to `NA`) those datapoints in the same way as we do with non-pred-corrected data. The warning will then be correct so we can leave that unchanged. To make this work I made the core pred-correction calculation only use the non-NA data.

I also rephrased the warning slightly: "cannot use the pred-corr VPC with censored data" is worded somewhat strongly... IMO one just has to be aware of what is shown in the plot. I also removed a duplicate of the warning that was commented out before.